GliaWidgets/Theme/Theme+Chat.swift

```diff
@@ -99,6 +99,7 @@ extension Theme {
             normalBackgroundColor: Color.lightGrey,
             highlightedTextColor: color.baseLight,
             highlightedBackgroundColor: color.primary,
+            disabledTabsBorderColor: Color.baseShade,
```

Comment:
"Tabs" is not used anywhere else in the code. They are instead called "options" :)
Also, this is an initializer for the choice card options, thus it is obvious that every parameter is related to it, so even "options" in the name is redundant.

Therefore I suggest renaming it to `disabledBorderColor`. It falls in line nicely with other naming schemes - state (normal/highlighted/disabled, what element it is related to (text/border/background) and what property (color).

**Also**, I suggest moving it to the end of the initializer to have a set order of text - background - border (more in-depth reasons in next comments).